libs/langchain-openai/src/chat_models.ts

```diff
@@ -912,8 +921,7 @@ export class ChatOpenAI<
       const params = {
         ...this.clientConfig,
         baseURL: endpoint,
-        timeout: this.timeout,
-        maxRetries: 0,
```

Comment:
We still need this to be 0 to avoid using the default SDK retry policy. We retry using the more generalized `AsyncCaller`

Reply:
Would be good to add a note here

Reply:
Done